README.md

```diff
@@ -450,6 +450,82 @@ with patch_database_model(Org, models=orgs), patch_database_model(
     assert org_get.tech_owner.username == "bberkley"
 ```
 
+### Type checking and mypy support
+
+Pynocular is fully type hinted and supports mypy in a limited way. Using mypy
+with the Pydantic mypy plugin will provide type checking for Pynocular database
+models' usual pydantic fields, but by default mypy will not know about the
+DatabaseModel specific methods.
+
+e.g.
+```python
+from pynocular.database_model import database_model
+from pydantic import BaseModel
+
+@database_model(...)
+class MyModel(BaseModel):
+  my_field: str = ...
+
+model = MyModel(...) # mypy understands this type
+print(model.my_field) # mypy will not error on this
+print(model.non_existent_field) # mypy will error here
+model_list = await MyModel.get_list(...) # mypy will (incorrectly) error here
+```
+
+The false positives are due to a limitation of python's type hint system that
+doesn't allow for dynamic class modification. The current workaround is to use
+the `BaseModel_database_model_hint` class provided by Pynocular, which is
```

Comment:
That class name doesn't follow Python convention and is also very wordy. We shouldn't have to do this in order get mypy to work. If the issue is the dynamic class modification we should look into fixing that first before patching over it